lib/src/messaging.dart

```diff
@@ -16,6 +16,8 @@ class Messaging extends JsObjectWrapper<messaging_interop.MessagingJsImpl> {
     return _expando[jsObject] ??= Messaging._fromJsObject(jsObject);
   }
 
+  static bool isSupported() => messaging_interop.isSupported();
```

Comment:
Did you test this locally?

Reply:
I did test this locally on the computer as well as on my hosted web app on an iPad where I got true and false respectively.